.github/dependabot.yml

```diff
@@ -10,7 +10,7 @@ updates:
     open-pull-requests-limit: 10
     schedule:
       interval: "daily"
-    target-branch: "master"
+    target-branch: "develop"

```

Comment:
why is this changed to develop?

Reply:
@rrawther  because there is no master branch for rocDecode. The above change is for building the doc based on the develop branch.